src/Parser/MarksmanshipHunter/Modules/Features/AimedInVulnerableTracker.js

```diff
@@ -15,41 +15,41 @@ class AimedInVulnerableTracker extends Module {
     enemies: Enemies,
   };
   totalAimed = 0;
-  inVulnAimed = 0;
-  outsideVulnAimed = 0;
+  inVulnerabilityAimed = 0;
+  outsideVulnerabilityAimed = 0;
 
   on_byPlayer_cast(event) {
     if (event.ability.guid !== SPELLS.AIMED_SHOT.id) {
       return;
     }
     const enemy = this.enemies.getEntity(event);
     if (enemy.hasBuff(SPELLS.VULNERABLE.id, event.timestamp)) {
-      this.inVulnAimed += 1;
+      this.inVulnerabilityAimed += 1;
     }
-    if (!enemy.hasBuff(SPELLS.VULNERABLE.id, event.timestamp)) {
-      this.outsideVulnAimed += 1;
+    else {
+      this.outsideVulnerabilityAimed += 1;
     }
     this.totalAimed += 1;
   }
   suggestions(when) {
-    const aimedOutsideVuln = this.outsideVulnAimed;
-    when(aimedOutsideVuln).isGreaterThan(0)
+    const percentAimedOutsideVulnerable = 100/this.totalAimed * this.outsideVulnerabilityAimed;
```

Comment:
Some weird arithmetic here, if you rearrange the terms you get `(this.outsideVulnerabilityAimed / this.totalAimed) * 100` which would be a percentage between 0 - 100% which means you gave it tolerance of 0,02% instead of 2%. Furthermore you divide or multiply by 100 further down the code. I suggest you change this to just `const percentAimedOutsideVulnerable = this.outsideVulnerableAimed / this.totalAimed` and remove the `* 100` and `/ 100` further down. Nice jobs on the added SpellLink :)

Reply:
I see you've updated the suggestion to be accurate but I still think using the decimal representation of percentage (e.g. 0.02 instead of 2) is more consistent with the rest of the code and allows for using `formatPercentage`.